projects/kit/components/progress-bar/progress-bar.component.less

```diff
@@ -0,0 +1,72 @@
+@import 'taiga-ui-local';
+
+.clearProgress() {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    border: none;
+}
+
+/* Used to style outer progress bar (container) */
+.progressContainer(@property, @value) {
+    @{property}: @value; // IE | Edge | Mozilla
+
+    &::-webkit-progress-bar {
+        @{property}: @value; // Chrome | Opera | Safari
+    }
+}
+
+/* used to style inner value bar */
+.progressValue(@property, @value) {
+    @{property}: @value; // Edge
+
+    &::-webkit-progress-value {
+        @{property}: @value; // Chrome | Opera | Safari
+    }
+
+    &::-moz-progress-bar {
+        @{property}: @value; // Mozilla
+    }
+}
+
+/*
+* Edge and IE animate the progress bar right out of the box
+* Chrome | Opera | Safari animation is controlled by its mixins
+* No possibility for firefox animation (https://snook.ca/archives/html_and_css/animating-progress)
+*/
+.progressAnimation() {
+    &::-webkit-progress-value {
+        transition: width var(--tui-duration) linear;
+    }
+}
+
+.progressValueColor() {
+    color: var(--tui-progress-color-old-edge); // IE | Not Chromium Edge
+
+    &::-webkit-progress-value {
+        background: var(--tui-progress-color); // Chromium Edge | Chrome | Opera | Safari
+    }
+
+    &::-moz-progress-bar {
+        background: var(--tui-progress-color); // Mozilla
+    }
+}
+
+:host {
+    .clearProgress();
+    .progressAnimation();
+    .progressValueColor();
+    .progressContainer(background-color, var(--tui-base-03));
+
+    border-radius: var(--tui-radius-xs);
+    overflow: hidden;
+
+    &[data-tui-host-size='m'] {
```

Comment:
Let's not use `-tui-host` anymore, just `data-size`. It just bloats CSS and HTML :)

Reply:
fixed